src/lib/factories.js

```diff
@@ -46,7 +46,7 @@ export function createShorthand(Component, mapValueToProps, val, defaultProps =
   } else if (_.isPlainObject(val)) {
     type = 'props'
     usersProps = val
-  } else if (_.isString(val) || _.isNumber(val)) {
+  } else if (_.isString(val) || _.isNumber(val) || _.isArray(val)) {
```

Comment:
To give a little more context, I came up with these shorthands by building upwards and trying to collapse where possible. So:
```js
// This
<Table.Cell>Name</Table.Cell>
// became
<Table.Cell content='Name' />


// This
<Table.Row>
  <Table.Cell content='Name' />
  <Table.Cell content='Status' />
  <Table.Cell content='Company' />
</Table.Row>

// became
<Table.Row cells=['Name', 'Status', 'Company'] />


// This
<Table>
  <Table.Header>
    <Table.Row cells=['Name', 'Status', 'Company'] />
  </Table.Header>
</Table>

// became
<Table headerRow={{ cells: ['Name', 'Status', 'Company'] }} />
```

If you think about our shorthand, generally it lets you pass a literal in place of a one-key object, e.g.
```js
// One-key object
<Label detail={{ content: 'Some detail' }} />

// Literal
<Label detail={'Some detail'} />
```

So I figured if we can describe a Table.Row as a one-key object, we should be able to describe it as a "literal":
```js
// One-key object
<Table headerRow={{ cells: ['Name', 'Status', 'Company'] }} />

// Literal
<Table headerRow={['Name', 'Status', 'Company']} />
```

So I'm not thinking about using arrays to let you build an array of some component (e.g. `const buttons = Button.create(['A', 123])`), but instead I'm thinking about it as a valid way of describing a component that has a list of things (e.g. `TableRow` has a list of cells that can completely describe that row).

Other examples:
```js
// One-key object
Menu.create({ items: ['menu item 1', 'menu item 2', 'menu item 3'] })

// Literal
Menu.create(['menu item 1', 'menu item 2', 'menu item 3'])


// One-key object
List.create({ items: [<List.Item />, { content: 'item 2', description: 'item 2'}, 'list item 3'] })

// Literal
List.create([<List.Item />, { content: 'item 2', description: 'item 2'}, 'list item 3'])
```

Thoughts?